pages/about/staff.vue

```diff
@@ -196,17 +196,48 @@ export default {
                     { name: 'Zoey', photo: 'default.jpg' },
                 ],
                 finance: [{ name: 'Rock', photo: 'finance/Rock.jpg' }],
-                reviewer: [],
+                reviewer: [
+                    { name: 'Killik Kuo', photo: 'default.jpg' },
+                    { name: 'Tenglin Yu', photo: 'reviewer/Tenglin Yu.jpg' },
+                    { name: 'Max Lai', photo: 'default.jpg' },
+                    { name: '黃坤賢', photo: 'default.jpg' },
+                    { name: 'Po-Chun Lu', photo: 'reviewer/Pochun.jpeg' },
+                    { name: 'Wei Lee', photo: 'default.jpg' },
+                    { name: 'JunWei Song', photo: 'reviewer/JunWei Song.jpeg' },
+                    { name: 'Chihsuan Huang', photo: 'default.jpg' },
+                    { name: 'Kir', photo: 'reviewer/Kir.jpg' },
+                    { name: '曾君宇', photo: 'default.jpg' },
+                    { name: 'Jian-Ming Huang', photo: 'default.jpg' },
+                    { name: 'Stanley Huang', photo: 'default.jpg' },
+                    { name: 'Yu-Hsun Lee', photo: 'reviewer/Yu Hsun Lee.jpg' },
+                    { name: 'Wei Lin', photo: 'reviewer/Wei Lin.jpg' },
+                    { name: '曲華榮', photo: 'reviewer/petertc.jpeg' },
+                    { name: 'Keith Yang', photo: 'default.jpg' },
+                    { name: '郭學聰(橘向日葵)', photo: 'default.jpg' },
+                    { name: 'Shuhsi Lin', photo: 'default.jpg' },
+                    { name: 'Sosorry', photo: 'default.jpg' },
+                    { name: 'Candy', photo: 'default.jpg' },
+                    { name: 'Louie Lu', photo: 'default.jpg' },
+                    { name: 'Yuan Chao', photo: 'reviewer/Yuan Chao.jpg' },
+                    { name: '阿哲', photo: 'default.jpg' },
+                    { name: 'Daisuke Saito', photo: 'default.jpg' },
+                    { name: 'Adrian Liaw', photo: 'default.jpg' },
+                    { name: 'Zong-Han Xie', photo: 'default.jpg' },
+                    { name: 'Tzu-Ping Chung', photo: 'default.jpg' },
+                    {
+                        name: 'Cheng-Lung Sung',
+                        photo: 'reviewer/Cheng-Lung Sung.jpg',
+                    },
+                    { name: 'Kuo-Tung Kao', photo: 'default.jpg' },
+                    { name: 'Maurice Ling', photo: 'default.jpg' },
+                    { name: 'KK', photo: 'default.jpg' },
+                    { name: 'Chihchun Tsai', photo: 'default.jpg' },
+                    { name: '陳俊嘉 ccc', photo: 'default.jpg' },
+                ],
                 photography: [
                     { name: 'Andy', photo: 'photography/Andy.jpg' },
                     { name: 'Deimos', photo: 'photography/Deimos.jpg' },
-                    { name: 'Charles', photo: 'photography/Charles.jpg' },
-                    { name: '孝玠', photo: 'photography/孝玠.jpg' },
-                    { name: '庭蓁', photo: 'photography/庭蓁.jpg' },
```

Comment:
Why were items of photography deleted?

Reply:
Oh...it's a mistake, i already make it right. Thanks.